docs/PERFORMANCE_BASELINE.md

```
@@ -0,0 +1,276 @@
+| **Total Blocking Time**      | 230ms         | B+    | <300ms | <200ms |
+| **Speed Index**              | 1.2s          | A     | <3.4s  | <2.5s  |
+
+\*FID not measurable in synthetic testing environment
+
+---
+
+## 📦 Resource Budget Baseline
+
+### Bundle Size Analysis
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)